cl-postgres/tests/tests-scram.lisp

```diff
@@ -0,0 +1,170 @@
+;;;; -*- Mode: LISP; Syntax: Ansi-Common-Lisp; Base: 10; Package: CL-POSTGRES-TESTS; -*-
+(in-package :cl-postgres-tests)
+
+(def-suite :cl-postgres-scram
+    :description "Scram Test suite for cl-postgres"
+    :in :cl-postgres)
+
+(in-suite :cl-postgres-scram)
+
+;; From RFC 7677
+;;   This is a simple example of a SCRAM-SHA-256 authentication exchange
+;;    when the client doesn't support channel bindings.  The username
+;;    'user' and password 'pencil' are being used.
+
+;;    C: n,,n=user,r=rOprNGfwEbeRWgbNEkqO
+
+;;    S: r=rOprNGfwEbeRWgbNEkqO%hvYDpWUa2RaTCAfuxFIlj)hNlF$k0,s=W22ZaJ0SNY7soEsUEjb6gQ==,i=4096
+
+;;    C: c=biws,r=rOprNGfwEbeRWgbNEkqO%hvYDpWUa2RaTCAfuxFIlj)hNlF$k0,p=dHzbZapWIk4jUhN+Ute9ytag9zjfMHgsqmmiz7AndVQ=
+
+;;    S: v=6rriTRBi23WpRR/wtup+mMhUZUn/dB5nLTJRsjl95G4=
+
+(test string-to-usb-array
+  (is (equalp (cl-base64:base64-string-to-usb8-array
+              (cl-base64:string-to-base64-string "r=fyko+d2lbbFgONRv9qkxdawL3rfcNHYJY1ZVvWVs7j,s=QSXCR+Q6sek8bf92,i=4096"))
```

Comment:
Indentation seems off on this line - should be two spaces more.

Reply:
Agreed.